bugsnag-plugin-android-ndk/src/androidTest/java/com/bugsnag/android/ndk/migrations/EventMigrationV4Tests.kt

```diff
@@ -0,0 +1,182 @@
+package com.bugsnag.android.ndk.migrations
+
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotEquals
+import org.junit.Test
+
+class EventMigrationV4Tests : EventMigrationTest() {
+
+    @Test
+    /** check notifier and api key, since they aren't included in event JSON */
+    fun testMigrationPayloadInfo() {
+        val infoFile = createTempFile()
+
+        val info = migratePayloadInfo(infoFile.absolutePath)
+
+        assertEquals(
+            mapOf(
+                "apiKey" to "5d1e5fbd39a74caa1200142706a90b20",
+                "notifierName" to "Test Library",
+                "notifierURL" to "https://example.com/test-lib",
+                "notifierVersion" to "2.0.11"
```

Comment:
This info is persisted but currently wholly unused - this key _should_ be used to send the event, but is not afaict.